reconcile/vpc_peerings_validator.py

```diff
@@ -14,10 +16,82 @@
 )
 from reconcile.status import ExitCodes
 from reconcile.utils import gql
+from reconcile.utils.aws_api import AWSApi
 
 QONTRACT_INTEGRATION = "vpc-peerings-validator"
 
 
+def validate_no_cidr_overlap(
+    query_data: VpcPeeringsValidatorQueryData,
+) -> bool:
+    clusters: list[ClusterV1] = query_data.clusters or []
+
+    for cluster in clusters:
+        peerings_entries = [
+            {
+                "provider": "cluster-self-vpc",
+                "vpc_name": cluster.name,
+                "cidr_block": cluster.network.vpc,  # type: ignore[union-attr]
+            },
+        ]
+        if cluster.peering:
```

Comment:
init peerings_entries only have peering info. avoid issues in case some external clusters don't have vpc info.
```suggestion
        if cluster.peering:
            peerings_entries = [
                {
                    "provider": "cluster-self-vpc",
                    "vpc_name": cluster.name,
                    "cidr_block": cluster.network.vpc,  # type: ignore[union-attr]
                },
            ]
```